src/runmode-pfring.c

```diff
@@ -264,10 +264,13 @@ static void *ParsePfringConfig(const char *iface)
                 }
             }
         } else {
-            if (StringParseUint16(&pfconf->threads, 10, 0, (const char *)threadsstr) < 0) {
+            uint16_t threads = 0;
```

Comment:
Out of curiosity, why not change the definition of `threads` in `PfringIfaceConfig` structure ?

Reply:
Would be good to do. This is just quick to get rid of the warning.

Reply:
Ok thanks